typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -272,8 +272,7 @@ export const hyperlaneContextAgentChainConfig: AgentChainConfig<
     moonbeam: true,
     morph: true,
     nero: true,
-    // At the moment, we only relay between Neutron and Manta Pacific on the neutron context.
-    neutron: false,
+    neutron: true,
```

Comment:
will this trigger keyfunder balance alerts?

Reply:
It did 😄 nice call, it's funded now